papers/bcr-2020-010-output-desc.md

```diff
@@ -288,20 +291,20 @@ D9 0193                                 # tag(403) public-key-hash
                   F4                    # primitive(20) is-hardened false
                   80                    # array(0) child-index-wildcard
                   F4                    # primitive(20) is-hardened false
-         08                             # unsigned(8) parent-fingerprint
+         08                             # unsigned(8) parent-fingerprint 
          1A 78412E3A                    # unsigned(2017537594)
 ```
 
 * As a hex string:
 
 ```
-D90193D9012FA503582102D2B36900396C9282FA14628566582F206A5DD0BCC8D5E892611806CAFB0301F0045820637807030D55D01F9A0CB3A7839515D796BD07706386A6EDDF06CC29A65A0E2906D90130A20186182CF500F500F5021AD34DB33F07D90130A1018401F480F4081A78412E3A
+D90193D9012FA503582102D2B36900396C9282FA14628566582F206A5DD0BCC8D5E892611806CAFB0301F0045820637807030D55D01F9A0CB3A7839515D796BD07706386A6EDDF06CC29A65A0E2906D90130A30186182CF500F500F5021AD34DB33F030407D90130A1018401F480F4081A78412E3A
 ```
 
 * As a UR:
 
 ```
-ur:crypto-output/taadmutaaddlonaxhdclaotdqdinaeesjzmolfzsbbidlpiyhddlcximhltirfsptlvsmohscsamsgzoaxadwtaahdcxiaksataxbtgotictnybnqdoslsmdbztsmtryatjoialnolweuramsfdtolhtbadtamtaaddyoeadlncsdwykaeykaeykaocytegtqdfhattaaddyoyadlradwklawkaycyksfpdmftpyaaeelb
+ur:crypto-output/fyesdyeheseofyesdyeheyfgfpecdyeoeceteyehdyeyfyeyfweoenesdydyeoesenfxeseyeteyfgfpeheeeneyeteceneneceteyfgeydyenfpecfyfydyfwfxfxetfyecfeeteseyenehehetdyenfxfpfgfwdyeodyehfgdydyeeeceteydyeneoemetdyemdyeodyfyececfydyehfgesfpdyfxfweofpemeteoesecehecfyemesenfwfydyememdyeneoetenfpenfefyfyfgdyenfxfxeyesfpenecfpdyfeeyesdyenfyesdyeheodyfpeodyeheteneheteyfxfgecdydyfgecdydyfgecdyeyehfpfyeoeefyfweoeofgdyeodyeedyemfyesdyeheodyfpehdyeheteedyehfgeeetdyfgeedyetehfpemeteeeheyfeeofpgyvdhfva
```

Comment:
cbor looks good. 

However for UR i get:

```
$ bytewords -i hex D90193D9012FA503582102D2B36900396C9282FA14628566582F206A5DD0BCC8D5E892611806CAFB0301F0045820637807030D55D01F9A0CB3A7839515D796BD07706386A6EDDF06CC29A65A0E2906D90130A30186182CF500F500F5021AD34DB33F030407D90130A1018401F480F4081A78412E3A  -o minimal
taadmutaaddlonaxhdclaotdqdinaeesjzmolfzsbbidlpiyhddlcximhltirfsptlvsmohscsamsgzoaxadwtaahdcxiaksataxbtgotictnybnqdoslsmdbztsmtryatjoialnolweuramsfdtolhtbadtamtaaddyotadlncsdwykaeykaeykaocytegtqdfhaxaaattaaddyoyadlradwklawkaycyksfpdmftkiiozsfd
```

Can you fix this and squash your commits?

Reply:
> Can you fix this and squash your commits?

Will do. 

I was using [foundation-ur](https://github.com/Foundation-Devices/foundation-ur-py). I'm not sure why I was getting a different output. I'll update after I understand what I was doing wrong.